asllib/doc/ASLSemanticsReference.tex

```diff
@@ -191,17 +919,76 @@ \chapter{Evaluation of Expressions \label{chap:eval_expr}}
 \item SemanticsRule.CTC (see Section~\ref{sec:SemanticsRule.CTC})
 \end{itemize}
 
+\section{Evaluating Lists of Expressions}
+
+We define two helper relations for evaluating a list of expressions:
+\[
+  \begin{array}{rcl}
+    \texttt{eval\_expr\_list}     &:& (\envs \times \expr^*) \aslrel (\Normal((\vals^* \times \XGraphs), \envs) \cup \TThrowing \cup \TError)\\
+    \texttt{eval\_expr\_list\_m}  &:& (\envs \times \expr^*) \aslrel \\
+                                  & & (\Normal((\vals\times\XGraphs)^* \times \envs) \cup \TThrowing \cup \TError)\\
```

Comment:
Right. This may be confusing but when I define the types of configurations like M(a, b, c), in "OCaml-style Notations", then the commas are interpreted as \times.

Reply:
Between the elements of a configuration the comma means \times, but otherwise - inside elements of configurations - I haven't introduced any special meaning so I have to use \times. I had a feeling this would be confusing. Notice that I never do M(a \times b), only things like M(a, (b\times c)).

Reply:
Perhaps I should explain this ahead of time and give an example. The thing is, the commas add more structure to the type definitions, otherwise it's just a long list of \times.

Reply:
This is a consistency error.

Reply:
I think I managed to catch all those problems now.